engine/access/rpc/backend/backend_scripts.go

```diff
@@ -25,7 +26,12 @@ type backendScripts struct {
 	state             protocol.State
 	connFactory       ConnectionFactory
 	log               zerolog.Logger
-	seenScripts       map[[md5.Size]byte]time.Time // to keep track of unique scripts sent by clients. bounded to 1MB (2^16*2*8) due to fixed key size
+	seenScripts       scriptMap
```

Comment:
Should this be a pointer?
```suggestion
	seenScripts       *scriptMap
```